File: week-02/day-01/cuboid.py
# Write a program that stores 3 sides of a cuboid as variables (doubles)
# The program should write the surface area and volume of the cuboid like:
# 
# Surface Area: 600
# Volume: 1000
height = 600
width = 600
length = 600

surfaceArea = (height * width + width * length + length * height)
volume = (height * width * length)
print(surfaceArea)
print(volume)

File: week-07/day-02/cuboid.js
'use strict';

// Write a program that stores 3 sides of a cuboid as variables (floats)
// The program should write the surface area and volume of the cuboid like:
//
// Surface Area: 600
// Volume: 1000

let a = 3.5;
let b = 4.5;
let c = 5.5;

console.log('volume '+ (a*b*c));
console.log('surface ' + 2*(a*b+a*c+b*c));